File: vb.net/send_single_message.vb
Imports System.Net.Http
Imports System.Text
Imports System.Text.Json

Module Module1
    Async Function SendSms() As Task
        Dim apiUrl As String = "<REPLACE_WITH_PROD_URL>/v1/messages/send"
        Dim token As String = "<REPLACE_WITH_YOUR_API_KEY>"

        Using httpClient As New HttpClient()
            httpClient.DefaultRequestHeaders.Add("Authorization", $"Bearer {token}")

            Dim requestData = New With {
                .from = "SMS INFO",
                .to = "2250000000000",
                .content = "Hello API!",
                .dlrUrl = "https://mydomain.com:4444/dlr",
                .dlrMethod = "GET",
                .customData = "customData",
                .sendAt = "2023-02-13T21:40:00.000Z"
            }

            Dim json As String = JsonSerializer.Serialize(requestData)
            Dim content As New StringContent(json, Encoding.UTF8, "application/json")

            Dim response As HttpResponseMessage = Await httpClient.PostAsync(apiUrl, content)
            Dim responseBody As String = Await response.Content.ReadAsStringAsync()

            Console.WriteLine(responseBody)
        End Using
    End Function

    Sub Main()
        SendSms().Wait()
    End Sub
End Module

File: php/guzzle/send_single_message.php
<?php

use GuzzleHttp\Client;
use GuzzleHttp\Psr7\Request;

$prodUrl = '<REPLACE_WITH_PROD_URL>';
$token = '<YOUR_TOKEN>';

$data = [
    'from' => 'SMS INFO',
    'to' => '2250000000000',
    'content' => 'Hello API!',
    'dlrUrl' => 'https://mydomain.com:4444/dlr',
    'dlrMethod' => 'GET',
    'customData' => 'customData',
    'sendAt' => '2023-02-13T21:40:00.000Z'
];

$client = new Client([
    'headers' => [
        'Authorization' => 'Bearer ' . $token,
        'Content-Type' => 'application/json',
        'Accept' => 'application/json'
    ]
]);

try {
    $response = $client->post($prodUrl . '/v1/messages/send', [
        'json' => $data
    ]);
    
    echo $response->getBody();
} catch (\GuzzleHttp\Exception\RequestException $e) {
    echo 'Error: ' . $e->getMessage();
    if ($e->hasResponse()) {
        echo ' - ' . $e->getResponse()->getBody();
    }
}